### PR TITLE
Create 2022 election docs

### PR DESCRIPTION
**Pull request by christopherclark**
Initial docs for 2022 election. This reflects almost the same dates as 2021 - if the TOC wants to amend anything at all here, please do. Before May 11, I'll have calculated a list of eligible voters, and drafted a cf-dev post.

### CODE REVIEW

File: toc/elections/2022/Readme.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Voting Process
+
+
+The election will be conducted using a time-limited [Condorcet](https://civs.cs.cornell.edu/rp.html) ranking 
+on [CIVS](http://civs.cs.cornell.edu/) using the Schulze method. 
+
+
+Voters will receive an email with a ballot link. Voters will have until the end of the election cycle 
+to submit their ballot.
```
Comment (emalm) [inline toc/elections/2022/Readme.md:66]: I'd like us to explore using https://github.com/elekto-io/elekto as the election system for this cycle, which Kubernetes has been using for its [Steering Committee elections](https://groups.google.com/g/kubernetes-dev/c/Ds-A499pzg4) and which Knative also [started using last year](https://knative.dev/blog/steering/toc-2021-election-announcement/). We ran into significant confusion with the email exchanges and the once-a-day syncing cycle from the CIVS service last time.

Comment (emalm) [reply toc/elections/2022/Readme.md:66]: BTW, I started getting Elekto to build and run on CF on my fork at https://github.com/emalm/elekto/tree/run-with-cf. I've gotten it to stage and to connect to a bound MySQL DB automatically, but I haven't tried configuring the GitHub settings yet.

Comment (emalm) [reply toc/elections/2022/Readme.md:66]: Hah, I kept playing around with this, and it's now running at https://elekto-emalm.apps.dhaka.cf-app.com, with the election config repo at https://github.com/emalm/elekto.meta.test. @christopherclark, I made you both an admin and a voter for the "name the app" election that it's running right now, if you wanted to take a look at it.

**Review by emalm — APPROVED** [review]
LGTM. We'll plan to evaluate Elekto soon and to amend this as needed before voting starts if we want to proceed with it.

**Review by emalm — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Review by loewenstein — APPROVED** [review]

**Comment by emalm** [comment]
Merging, as we have a TOC quorum of approvals and it's been out for over a week without changes (other than to move the dates forward as needed).

**Pull request merged**